test/e2e-params.test.ts

```diff
@@ -77,5 +74,5 @@ describe('E2E - Params', () => {
     request(server)
       .get('/api/user/2/lastName')
       .expect(200)
-      .then(res => expect(res.body).toStrictEqual('Liskov')));
+      .then(res => expect(res.body).toStrictEqual({ lastName: 'Liskov' })));
```

Comment:
a bit off-topic but I just noticed this now because of the changes made here. 

What exactly are we testing here? I think this test pretty doesn't cover anything new that the test above this one already does. Or am I missing something?

Reply:
It's just to test Nth sub routes.

Reply:
Okay makes sense. In that cases my only suggestion is to replace `{ lastName: 'Liskov' }` with `{ lastName: DATA[1].lastName}` to make sure if we change Data we do not need to change this test (and to be in line with the above assertions.